libzecale/tests/aggregator/aggregator_test.cpp

```diff
@@ -299,7 +302,7 @@ void aggregator_test()
     std::cout << "[DEBUG] nested_proofs[0].get_primary_inputs().size(): "
               << batch[0]->get_primary_inputs().size() << std::endl;
     // Make sure that we have the right amount of primary inputs
-    ASSERT_EQ(batch[0]->get_primary_inputs().size(), 9);
+    ASSERT_EQ(batch[0]->get_primary_inputs().size(), num_zeth_inputs);
```

Comment:
`num_zeth_inputs` this could now become a protocol constant since we will assume that the nested applications apply GGPR's trick on their instances.

Reply:
I'm not sure I get exactly how this would look.  Where would this constant be defined?  It seems like a property of the aggregator_server (outside the scope of this test), rather than the (currently generic) aggregator circuit.
Unless we remove the `num_nested_inputs` parameter to the aggregator circuit and only allow it to be instantiated with `num_nested_inputs = 1`.

Reply:
> Where would this constant be defined? It seems like a property of the aggregator_server (outside the scope of this test), rather than the (currently generic) aggregator circuit.

Yes indeed, my bad. I lost track of the context here (we're not in the `aggregator_server`...)